src/Php/PhpConverter.php

```diff
@@ -119,34 +119,36 @@ private function visitTypeBase(PHPClass $class, Type $type)
             $this->visitComplexType($class, $type);
         }
     }
-	
+
 	/**
 	 * Process xsd:complexType xsd:sequence xsd:element
 	 *
 	 * @param PHPClass $class
 	 * @param Schema $schema
 	 * @param Sequence $sequence
 	 */
-	private function visitSequence(PHPClass $class, Schema $schema, Sequence $sequence)
-	{
+	private function visitSequence(PHPClass $class, Schema $schema, Sequence $sequence): void
+    {
 		foreach ($sequence->getElements() as $childSequence) {
-			if ($childSequence instanceof Group) {
+            if ($childSequence instanceof Group) {
```

Comment:
indentation seems weird...

Reply:
Why, I don't see anything wrong with it - shows as exactly the same indent as before?
Or were there spaces _and_ tabs used before?

Reply:
yeah, there where mixed spaces and tab indents before, I converted all to spaces now. as it should be.